app/models/user_setting.rb

```diff
@@ -1,3 +1,6 @@
 class UserSetting < ApplicationRecord
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/Documentation
**Severity**: INFO
**File**: app/models/user_setting.rb L1

Missing top-level class documentation comment.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*